docs/howtos/cert-manager.md

```diff
@@ -47,75 +55,74 @@ The DNS-01 challenge verifies domain ownership by proving you have control over
 
 1. Create the IAM policy specified in the cert-manager [AWS Route53](https://cert-manager.readthedocs.io/en/latest/tasks/acme/configuring-dns01/route53.html) documentation.
 
-2. Note the `accessKeyID` and create a secret named `prod-route53-credentials-secret` holding the `secret-access-key`.
+2. Note the `accessKeyID` and create a `secret` named `prod-route53-credentials-secret` in the cert-manager namespace that has a key value: `secret-access-key` from your AWS IaM credentials.
 
-3. Create and apply a `ClusterIssuer`:
+3. Create and apply a `ClusterIssuer`.  Make sure that the `Issuer` and the `secret` are in the same namespace.
 
     ```yaml
     ---
     apiVersion: cert-manager.io/v1alpha2
     kind: ClusterIssuer
     metadata:
       name: letsencrypt-prod
-      namespace: default
     spec:
       acme:
-        email: example@example.com
+        email: email@example.com
         server: https://acme-v02.api.letsencrypt.org/directory
         privateKeySecretRef:
           name: letsencrypt-prod
-        dns01:
-          providers:
-          - name: route53
+        solvers:
+        - selector:
+            dnsZones:
+              - "myzone.route53.com"
+          dns01:
             route53:
               region: us-east-1
-              accessKeyID: {SECRET_KEY}
+              accessKeyID: {accessKeyID}
+              hostedZoneID: {Hosted Zone ID} # optional, allows you to reduce the scope of permissions in Amazon IAM
               secretAccessKeySecretRef:
-                name: prod-route53-credentials-secret
+                name: prod-route54-credentials-secret
                 key: secret-access-key
     ```
 
-4. Create and apply a certificate:
-
+4. Create and apply a `Certificate`.  Make sure the `Certificate` is in the same namespace as Ambassador.
 
     ```yaml
     ---
     apiVersion: cert-manager.io/v1alpha2
     kind: Certificate
     metadata:
       name: ambassador-certs
-      namespace: default
+      # cert-manager will put the resulting Secret in the same Kubernetes namespace
+      # as the Certificate. Therefore you should put this Certificate in the same namespace as Ambassador.
+      # eg. if you deploy ambassador to ambassador namespace, you need to change to namespace: ambassador
+      namespace: ambassador
     spec:
+      # naming the secret name certificate ambassador-certs is important because
+      # ambassador just look for this particular name
```

Comment:
This has not been true for a long time. Where are you getting this information?

**Edit:** I see this was added to the HTTP-01 challenge yaml so you just made sure it was repeated here. Someone made a mistake merging that change in when it was so I am removing both.